library.go

```diff
@@ -1793,8 +1793,7 @@ func string_split(fnname string, recv_ Value, args Tuple, kwargs []Tuple) (Value
 		} else if fnname == "split" {
 			res = splitspace(recv, maxsplit+1)
 		} else { // rsplit
-			// TODO(adonovan): implement.
-			return nil, fmt.Errorf("rsplit(None, %d): maxsplit > 0 not yet supported", maxsplit)
+			res = rsplit(recv, maxsplit)
```

Comment:
> I don't know where a good place for unit tests would be but I would like to add some.

The dynamic tests live in the testdata/*.sky files.  In testdata/string.sky, you'll find the existing tests for rsplit.  You can put addition checks for rsplit there.